### PR TITLE
[CMSP-532] Drop `pre-ga/` from WordPress (Composer Managed) guide & update paths

### PR DESCRIPTION
**Pull request by jazzsequence**
## Summary

**[WordPress with Composer on Pantheon](https://docs.pantheon.io/guides/wordpress-composer/pre-ga/wordpress-composer-managed)** - Moves this guide out of the `/pre-ga` path and some minor updates to the content. Also updates all incoming links to that path to remove `pre-ga`.

### Dependencies and Timing

**Release**:
- [x] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/old-path/` => `/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)

### CODE REVIEW

**Review by pwtyler — APPROVED** [review]

**Comment by jazzsequence** [comment]
@stevector Anything special need to be done here to change the path of this doc?

**Comment by stevector** [comment]
@jazzsequence, @rachelwhitton would know more about the redirects. It looks like there is already a redirect in the other direction

```
curl -I https://docs.pantheon.io/guides/wordpress-composer/wordpress-composer-managed
HTTP/2 301
server: Varnish
retry-after: 0
location: https://docs.pantheon.io/guides/wordpress-composer/pre-ga/wordpress-composer-managed
```

**Comment by stevector** [comment]
@jazzsequence can we also delete "Use Integrated Composer with WordPress" https://pr-9002-documentation.appa.pantheon.site/guides/wordpress-composer/wordpress-ic

I don't know the situation in which a team should follow the guidance there instead of the guidance in https://pr-9002-documentation.appa.pantheon.site/guides/wordpress-composer/wordpress-composer-managed

File: source/content/guides/autopilot/07-troubleshoot.md
Comment (stevector) [inline source/content/guides/autopilot/07-troubleshoot.md:376]: I know you were just updating URLs, but can this sentence be rewritten to make sense?

File: source/content/guides/integrated-composer/01-introduction.md
Comment (stevector) [inline source/content/guides/integrated-composer/01-introduction.md:46]: @jazzsequence The section that this line appears in is very confusing. Does the removal of the pre-ga URL also mean we are getting rid of the documentation on https://github.com/pantheon-upstreams/wordpress-project

**Review by stevector — CHANGES_REQUESTED** [review]
I don't know if it all needs to happen in one PR, but I'd like to steer people toward one clear path for Integrated Composer + WordPress. Can we get rid of https://pr-9002-documentation.appa.pantheon.site/guides/wordpress-composer/wordpress-ic now?

File: source/content/guides/integrated-composer/01-introduction.md
Comment (jazzsequence) [reply source/content/guides/integrated-composer/01-introduction.md:46]: We aren't getting rid of `wordpress-project` until WordPress Composer Managed is supported by build tools. Today it is not.

**Comment by jazzsequence** [comment]
>  can we also delete "Use Integrated Composer with WordPress" https://pr-9002-documentation.appa.pantheon.site/guides/wordpress-composer/wordpress-ic

We can, but this doc is largely about forking the repository (e.g. for your own custom upstream). For sure it needs to be updated but that is outside the scope of [the issue](https://getpantheon.atlassian.net/browse/CMSP-532) which is just to remove the `/pre-ga` subdirectory.

File: source/content/guides/autopilot/07-troubleshoot.md
Comment (jazzsequence) [reply source/content/guides/autopilot/07-troubleshoot.md:376]: That document is owned by @pantheon-systems/lifecycle-ops (Autopilot). I am not sure the context of the note that was added.

**Comment by stevector** [comment]
> that is outside the scope of [the issue](https://getpantheon.atlassian.net/browse/CMSP-532) which is just to remove the /pre-ga subdirectory.

Is there more Engineering work coming soon that will lead to cleaning up the other WordPress + Composer pages?

File: source/content/guides/autopilot/07-troubleshoot.md
Comment (pwtyler) [reply source/content/guides/autopilot/07-troubleshoot.md:376]: AFAIK What it means is Build Tools sites do not work with Autopilot. If you are using Composer, you must use IC.

**Comment by rachelwhitton** [comment]
> @jazzsequence, @rachelwhitton would know more about the redirects. It looks like there is already a redirect in the other direction
> 
> ```
> curl -I https://docs.pantheon.io/guides/wordpress-composer/wordpress-composer-managed
> HTTP/2 301
> server: Varnish
> retry-after: 0
> location: https://docs.pantheon.io/guides/wordpress-composer/pre-ga/wordpress-composer-managed
> ```

I just removed that redirect in AGCDN. 

```
$ curl -I https://docs.pantheon.io/guides/wordpress-composer/wordpress-composer-managed
HTTP/2 200
```

Immediately after this PR is merged I can add the new redirect

**Comment by rachelwhitton** [comment]
>can we also delete "Use Integrated Composer with WordPress" https://pr-9002-documentation.appa.pantheon.site/guides/wordpress-composer/wordpress-ic

Removing `pre-ga` from the permalink also updates the lefthand navigation across the entire _WordPress with Composer on Pantheon_ "guide" (because they're grouped together in the file structure and the new permalink for the bedrock page now matches the permalink structure of the other pages) so I think it needs to be part of the scope of this issue to reconcile the single page as part of the whole guide.

Screenshot 2024-05-17 at 10 32 56 AM


If not, I think we should just convert the Bedrock page to a standalone doc like `docs.pantheon.io/bedrock` since it was on it's own before already?

**Comment by jazzsequence** [comment]
>  Removing pre-ga from the permalink also updates the lefthand navigation across the entire WordPress with Composer on Pantheon "guide" (because they're grouped together in the file structure and the new permalink for the bedrock page now matches the permalink structure of the other pages) so I think it needs to be part of the scope of this issue to reconcile the single page as part of the whole guide.

That is intentional. It's already numbered in such a way that it falls into the correct (or ideal) order for the guide. My hesitance about changing other (unrelated or tangentially related) docs in the entire guide is that a) updating _all_ of the WordPress Composer documentation was not scoped as part of the issue that we (CMSP) are working on and b) not _all_ of the WordPress Composer documentation is actually out-of-date _yet_. We are actively making changes that will continue to roll out but some of the old documentation and old ways of doing things are still _the only way to do it_ because their equivalents with the WordPress Composer Managed (Bedrock) upstream has not been built or tested.

File: source/content/guides/autopilot/07-troubleshoot.md
Comment (jazzsequence) [reply source/content/guides/autopilot/07-troubleshoot.md:376]:
```suggestion
A Composer-based WordPress site using [Build Tools](https://docs.pantheon.io/guides/build-tools/) is not supported by Autopilot.
```

@pwtyler this look good?

**Comment by jazzsequence** [comment]
With the exception of the [confusing line here](https://github.com/pantheon-systems/documentation/pull/9002#discussion_r1604069833) (which I've proposed an alternative to), is there anything blocking this from being merged @stevector @rachelwhitton ?

**Review by stevector — APPROVED** [review]

**Pull request merged**
